solutions/cpp/01-gg4/code/your_program.sh

```
@@ -22,4 +22,4 @@ set -e # Exit early if any commands fail
 #
 # - Edit this to change how your program runs locally
 # - Edit .codecrafters/run.sh to change how your program runs remotely
-exec $(dirname "$0")/build/git "$@"
+exec $(dirname $0)/build/git "$@"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)